components/ConditionalRender.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode, useEffect, useState } from 'react'
+
+export interface ConditionallyRenderProps {
```

Comment:
@xtealer can you please explain why you need this component? What is client vs server conditional render?

Reply:
This was a test to debug the stepper component in prod. Sometimes while using nextjs with SSR, the order of rendered html does not match when hydrated on client side browser. This is a workaround to allow SSR rendering only or client side rendering only. But newer versions of nextjs will support this natively so this component will become deprecated (maybe in a year or so).